src/screens/PasswordRecovery/componentes/EmailForm/index.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import {
+  KeyboardAvoidingView,
+  TextInput
+} from 'react-native'
+import * as Yup from 'yup'
+import { Formik } from 'formik'
+import { isEmpty, values } from 'lodash'
+import { Body, Description, GrayButton, PrimaryButton } from '~/components'
+import styles, { PLACEHOLDER_COLOR } from './styles'
+
+const initialValues = {
+  email: ''
+}
+
+const Error = ({ error }) => (
+  error && (
+    <Body style={styles.error}>
+      Email invalido
+    </Body>
+  )
+)
+
+const schema = Yup.object().shape({
+  email: Yup.string()
+    .email('Email inválido')
+    .required('Requerido')
+})
+
+const formHasBlankValues = (formValues) => {
+  const blankValues = values(formValues).filter(element => element === '')
+  return !isEmpty(blankValues)
+}
+
+const SubmitButton = ({ onPress, blankValuesExist, error, disabled, children }) => {
+  if (blankValuesExist || error) {
+    return (
+      <GrayButton style={styles.submitButton}>
+        <Description white>
+          {children}
+        </Description>
+      </GrayButton>
+    )
+  } else {
```

Comment:
@MPucheta is there a component that does this automatically when the `disabled` prop is true?

Reply:
Yes. I marked this suggestion in my review too, I also suggested a change regarding `disabled`